dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -533,22 +562,22 @@ else if (f.getFieldType().equals("date")) {
                     if(tagg.length() >tagDelimit.length()){
                     	String taggStr = tagg.substring(0, tagg.length()-tagDelimit.length());
                         m.put(st.getVelocityVarName() + "." + f.getVelocityVarName(), taggStr.replaceAll(",,", " "));
-                        m.put("tags", taggStr);
+                        m.put(ESMappingConstants.TAGS, taggStr);
                     }
 
 
                     if ( Structure.STRUCTURE_TYPE_PERSONA != con.getStructure().getStructureType() ) {
                         if ( personaTags.length() > tagDelimit.length() ) {
                         	String personaStr = personaTags.substring(0, personaTags.length()-tagDelimit.length());
-                            m.put(st.getVelocityVarName() + ".personas", personaStr);
-                            m.put("personas", personaStr);
+                            m.put(st.getVelocityVarName() + "."+ESMappingConstants.PERSONAS, personaStr);
+                            m.put(ESMappingConstants.PERSONAS, personaStr);
                         }
                     }
 
                 } else {
-                    if (f.getFieldContentlet().startsWith("bool")) {
+                    if (f.getFieldContentlet().startsWith(ESMappingConstants.FIELD_ELASTIC_TYPE_BOOLEAN)) {
                         m.put(st.getVelocityVarName() + "." + f.getVelocityVarName(), valueObj.toString());
-                    } else if (f.getFieldContentlet().startsWith("float") || f.getFieldContentlet().startsWith("integer")) {
+                    } else if (f.getFieldContentlet().startsWith(ESMappingConstants.FIELD_ELASTIC_TYPE_FLOAT) || f.getFieldContentlet().startsWith(ESMappingConstants.FIELD_ELASTIC_TYPE_INTEGER)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 196 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)